S/SLATEC/build_tarballs.jl

```diff
@@ -76,11 +91,10 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency(PackageSpec(name="LAPACK_jll", uuid="51474c39-65e3-53ba-86ba-03b1b862ec14")),
-    Dependency(PackageSpec(name="OpenBLAS_jll", uuid="4536629a-c528-5b80-bd46-f80d51c5b363")),
+    Dependency(PackageSpec(name="libblastrampoline_jll")),
     Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
 # For the time being need LLVM 11 because of <https://github.com/JuliaPackaging/BinaryBuilderBase.jl/issues/158>.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_llvm_version=v"11")
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.9", preferred_llvm_version=v"11")
```

Comment:
Why are we using llvm 11?

Reply:
Just left LLVM version as it was, because it was added by maintainers (maybe by you) in the current version. (The mentioned issue is of course closed 2 years ago.)
Should the LLVM version be removed? (I guess at least changed from 11, but should some other version be specified?)

BTW, is there somewhere list/table of recommended combinations of versions of at least main components that should be used with BinaryBuilder? I didn't find one at least on quick glance.